### PR TITLE
merge Invitation and DomainInvitation models

### PR DESCRIPTION
**Pull request by nickpell**
addresses https://github.com/dimagi/commcare-hq/pull/9223#issuecomment-158842139

### CODE REVIEW

**Comment by czue** [comment]
looks good!

File: corehq/apps/hqwebapp/utils.py [hunk 1]
```diff
@@ -192,7 +192,7 @@ def __call__(self, request, invitation_id, **kwargs):
                                                  password=form.cleaned_data["password"])
                     if authenticated is not None and authenticated.is_active:
                         login(request, authenticated)
-                    if isinstance(invitation, DomainInvitation):
+                    if isinstance(invitation, Invitation):
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/utils.py]: can't you get rid of these conditionals altogether?

**Comment by orangejenny** [comment]
hooray

File: corehq/apps/hqwebapp/utils.py (hunk 1 above)
Comment (nickpell) [inline corehq/apps/hqwebapp/utils.py]: yep: https://github.com/dimagi/commcare-hq/pull/9241

**Pull request merged**